code/modules/goonchat/browserOutput.dm

```diff
@@ -101,6 +101,8 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("data/iconCache.sav")) //Cache of ic
 		to_chat(owner, message)
 
 	messageQueue = null
+
+	sendMiningData()
 	sendClientData()
```

Comment:
We really need to modularize this at some point.

as more and more things slowly hook into the tgchat interface for general/generic client side code, this will become bloated and we'll wish we'd done it sooner.